src/components/Picker/Picker.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { Text } from '../Text';
+import { PICKER_DATA_HOOKS, PICKER_DATA_PROPS } from './dataHooks';
+import { ReactComponent as ChevronLeft } from '../../assets/icons/ChevronLeft.svg';
+import { ReactComponent as ChevronRight } from '../../assets/icons/ChevronRight.svg';
+import { IconButton } from '../IconButton';
+import { TPAComponentProps } from '../../types';
+import styles from './Picker.st.css';
+
+export interface PickerProps extends TPAComponentProps {
+  value: any;
+  onPrev(): void;
+  onNext(): void;
+  prevDisabled?: boolean;
+  nextDisabled?: boolean;
+}
+
+interface DefaultProps {
+  prevDisabled: boolean;
+  nextDisabled: boolean;
+}
+
+/** Picker */
+export class Picker extends React.Component<PickerProps> {
+  static defaultProps: DefaultProps = {
+    prevDisabled: false,
+    nextDisabled: false,
+  };
+
+  getDataAttributes() {
+    const { prevDisabled, nextDisabled } = this.props;
+
+    return {
+      [PICKER_DATA_PROPS.NextDisabled]: nextDisabled,
+      [PICKER_DATA_PROPS.PrevDisabled]: prevDisabled,
+    };
+  }
+
+  render() {
+    const {
+      onPrev,
+      value,
+      onNext,
+      prevDisabled,
+      nextDisabled,
+      ...rest
+    } = this.props;
+
+    return (
+      <div
+        data-hook={this.props['data-hook']}
+        {...styles('root', {}, rest)}
+        {...this.getDataAttributes()}
+      >
+        <IconButton
+          className={styles.arrow}
+          icon={<ChevronLeft />}
+          as="a"
```

Comment:
this shouldn't be used `as` an anchor.
It's not the correct usage here.
You can leave this without the `as` prop, for the default `button`